extensions/wikia/ArticleNavigation/scripts/sticky.js

```diff
@@ -88,13 +88,38 @@ require([
 		}
 	}
 
-	// Sticky navigation
-	stickyElementObject.init({
-		sourceElement: navigationElement,
-		alignToElement: boundBoxElement,
-		topFixed: globalNavigationHeight + additionalTopOffset,
-		minWidth: layout.breakpoints.smallMin,
-		adjustFunc: adjustValueFunction,
-		adjustPositionFunc: adjustPositionFunction
-	});
+	function adjustTopPosition() {
```

Comment:
As you're using ```navigationElement``` in ```isPositionStickySupported``` is it really needed to have separate function that is called only one time? Is there any more reason that I don't see here?